src/NzbDrone.Core.Test/ParserTests/AbsoluteEpisodeNumberParserFixture.cs

```diff
@@ -94,7 +94,7 @@ public class AbsoluteEpisodeNumberParserFixture : CoreTest
         [TestCase("Abc x Abc (2011) - 141 - Magician [KaiDubs] [1080p]", "Abc x Abc (2011)", 141, 0, 0)]
         [TestCase("Abc Abc 484 VOSTFR par Abc-Abc (1280*720) - version MQ", "Abc Abc", 484, 0, 0)]
         [TestCase("Abc - Abc Abc Abc - 107 VOSTFR par Fansub-Miracle Sharingan (1920x1080) - HQ_Draft", "Abc - Abc Abc Abc", 107, 0, 0)]
-        [TestCase("Abc Abc Abc Abc Episode 10 VOSTFR (1920x1080) Miracle Sharingan Fansub.MKV - Team - (� suivre)", "Abc Abc Abc Abc", 10, 0, 0)]
+        [TestCase("Abc Abc Abc Abc Episode 10 VOSTFR (1920x1080) Miracle Sharingan Fansub.MKV - Team - (À suivre)", "Abc Abc Abc Abc", 10, 0, 0)]
```

Comment:
Looks like this line can be reverted

Reply:
Sadly i don't even know why this was changed in the first place, looks like my IDE doesn't like the file encoding in there, does sonarr not use UTF8 by any chance? (i copy pasted the code over again and i don't see any changes in my git so i assume it just auto converts it in my IDE)